breadcord/core_modules/settings_manager/__init__.py

```diff
@@ -43,7 +44,7 @@ def __init__(self, bot: breadcord.Bot):
         super().__init__()
 
     async def on_submit(self, interaction: discord.Interaction) -> None:
-        with open(self.bot.settings_file, 'w', encoding='utf-8') as file:
+        async with aiofiles.open(self.bot.settings_file, 'w', encoding='utf-8') as file:
             file.write(self.editor.value)
```

Comment:
If we're switching to `aiofiles` here, would `file.write` not be a coroutine here, and therefore have to be awaited?

```suggestion
            await file.write(self.editor.value)
```

Reply:
Good catch